libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.html

```diff
@@ -7,6 +7,29 @@
   }"
 >
   <div class="sky-summary-action-bar-actions">
+    <div aria-live="polite">
+      @if (formErrors) {
+        @if (formErrors.length > 0) {
+          <div class="sky-summary-action-bar-errors sky-margin-stacked-lg">
+            @for (formError of formErrors; track formError.message) {
+              <div
+                [ngClass]="{
+                  'sky-margin-stacked-sm':
```

Comment:
Would putting this in css with a `:not(:last-of-type)` be more efficient instead of running javascript on every change detection cycle?